lib/iproov_flutter.dart

```diff
@@ -1,32 +1,64 @@
 import 'dart:async';
 import 'dart:convert';
 
-export 'package:iproov_flutter/options.dart';
+import 'package:flutter/services.dart';
+import 'package:iproov_flutter/events.dart';
+import 'package:iproov_flutter/options.dart';
+
 export 'package:iproov_flutter/enums.dart';
 export 'package:iproov_flutter/events.dart';
 export 'package:iproov_flutter/exceptions.dart';
+export 'package:iproov_flutter/options.dart';
 
-import 'package:flutter/services.dart';
-import 'package:iproov_flutter/options.dart';
-import 'package:iproov_flutter/events.dart';
+typedef IProovEventCallback = void Function(IProovEvent);
+
+const MethodChannel _iProovMethodChannel =
+    const MethodChannel('com.iproov.sdk');
+
+const EventChannel _iProovListenerEventChannel =
+    EventChannel('com.iproov.sdk.listener');
```

Comment:
Moved outside the main class. Same as in [camera](https://github.com/flutter/plugins/tree/master/packages/camera/camera) package.